### ISSUE TITLE
Use something better than 'cp' to update open parmlibs

### ISSUE THREAD

**Issue by nosrednayduj** [zowe/zowe-install-packaging#4127]
**Is your feature or enhancement request related to a problem or limitation? Please describe**

zwe component install uses UNIX "cp" to update the parmlib with the new ZWESIP00. If the data set is open, "cp" fails because it (stupidly) requires exclusive access.

**Describe your enhancement idea**

Long ago there was a CLIST-based mechanism for updating ZIS plug-in libraries. This was abandoned at some point during the 1.x development cycle. It would be great if this was brought back and used for all copies to data sets.

**Provide any additional context**

PR in which script that calls this was deleted: 
https://github.com/zowe/zowe-install-packaging/pull/965/commits/248f35af850ffef63620abe2f7717769282cd49a

Full file as before deleted:
https://github.com/zowe/zowe-install-packaging/blob/709cb254b38b17a94cc64fac4bb97cb44958312b/scripts/configure/zowe-copy-proc.sh

**Comment by JoeNemo** [comment]
Although the TSO method will work, there could be other OMVS ways of doing this.  Or programmable ways, which will definitely work but require more research and test.

**Comment by nosrednayduj** [comment]
Hey, no fair.  Just because an issue is not getting traction does not mean it should be auto-closed!

**Comment by JoeNemo** [comment]
Why do TSO CLISTs do better at obtaining the lock?  Do they wait and retry?

**Comment by JoeNemo** [comment]
Will SDSF or MXI show the specific ENQ conflict?

**Comment by nosrednayduj** [comment]
I don't have any idea about the internals. However, my experience is that cp NEVER allows you to put a member into a data set which is open. And the CLIST ALWAYS allows you to do it. I don't have any idea how any of this works; I just know that in Zowe 1.x I discovered this mechanism which was in use for updating the proclib, and I copied it for UMS purposes in updating proclib and all the other things that we update, like parmlibs. There is some drawback about creating a temporary data set in the users HLQ, and I think one customer tripped over that once.

**Comment by JoeNemo** [comment]
Sorry to disagree Judy, but I think that the activity of other users on the system is going to affect this behavior.   We probably need to rigorous test that guarantees that a freshly created PDS or PDSE that no one else has locked will exhibit the behavior you theorize.

**Comment by nosrednayduj** [comment]
Yes, I suppose my experience is "AnecData" and not necessarily true.

**Comment by JoeNemo** [comment]
Feel free to post more debugging data or MXI / SDSF screenshots when this happens.
